base/db/R/pft.add.spp.R

```diff
@@ -0,0 +1,121 @@
+#-------------------------------------------------------------------------------
```

Comment:
Git is still detecting this as an entirely new file, not that you moved the file from the scripts directory (it believes you deleted that file, not moved it). In the future PLEASE use git mv to move files, as we've now lost the revision history of this file. Understanding changes is also a lot harder this way as the file appears as if it's 100% new.

Reply:
got it!